src/scripts/admin/blocks/collapse/container/index.scss

```diff
@@ -174,7 +174,7 @@
   }
 
   .block-editor-block-list__block {
-    margin: 0;
+    // margin: 0;

```

Comment:
Kept commented to test result on staging. This may be added back, or removed in later commits.